griptape/drivers/__init__.py

```diff
@@ -242,4 +245,6 @@
     "OpenTelemetryObservabilityDriver",
     "GriptapeCloudObservabilityDriver",
     "DatadogObservabilityDriver",
+    "DreamMachineVideoGenerationDriver",
+    "BaseVideoGenerationDriver",
```

Comment:
am I the only one at this damned company that likes my lists alphabetized? Not for this PR, but dang

Reply:
I agree with james, but don't reorder this list in this PR. (Generally mixing refactoring or formatting with features makes it difficult for reviewers to see what is actually changing)

Reply:
`BaseVideoGenerationDriver` should be placed before `DreamMachineVideoGenerationDriver` for circular dependency (and alphabetical) reasons.